src/test/groovy/org/gradle/android/RoomSchemaLocationWorkaroundTest.groovy

```diff
@@ -7,6 +7,8 @@ import org.gradle.util.VersionNumber
 import spock.lang.Unroll
 
 class RoomSchemaLocationWorkaroundTest extends AbstractTest {
+    private static final String[] CLEAN_BUILD = ["clean", "testDebug", "testRelease", "--build-cache", "--stacktrace"]
```

Comment:
can you add `assembleAndroidTest` here? we should also make sure that the androidTest can compile since kapt also adds itself there.

Reply:
Good idea.  Done.